sprockets-media_query_combiner.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '>= 1.9.2'
 
-  gem.add_runtime_dependency "sprockets", "~>2.1"
```

Comment:
Actually, I was unable to launch Sprockets 4 on rails (it's in beta yet, and raises exceptions), but according to [this guide](https://github.com/rails/sprockets/blob/master/UPGRADING.md) I don't expect any issues running with 4.X version, thus I suggest putting `<5.0`.